lib/src/TailwindComponents/Button/index.module.scss

```
@@ -0,0 +1,354 @@
+:root {
+  --padding-inline: 0;
+  --padding-inline-has-icon: 0;
+}
+
+/* The 'components' layer is used so that the styles can be overridden by Tailwind's 'utilities' layer */
+@layer components {
+  .root {
+    position: relative;
+    overflow: hidden;
+    white-space: nowrap;
+    // text-overflow: ellipsis;
+    cursor: pointer;
+    border-style: solid;
+
+    /* VARIABLES FROM THEME - DONT CHANGE WITH REACT PROPS */
+    border-width: var(--border-width-button-default);
+    transition-property: all;
+    transition-duration: var(--transition-duration-button-default);
+    outline: transparent solid var(--outline-width-button-default);
+    font-weight: var(--font-weight-button-default);
+
+    /* VARIABLES THAT CHANGE WITH REACT PROPS */
+    height: var(--height);
+    padding-inline: var(--padding-inline);
+    border-radius: var(--border-radius-md);
+    font-size: var(--font-size);
+    color: var(--color-text-button-primary);
+    background-color: var(--color-background-button-primary-default);
+    border-color: var(--color-border-button-primary-default);
+
+    &:focus,
+    &:focus-visible {
+      outline-color: var(--color-outline-button-primary-focus);
+    }
+
+    @media (hover: hover) and (pointer: fine) {
+      &:hover {
+        background-color: var(--color-background-button-primary-hover);
+        border-color: var(--color-border-button-primary-hover);
+      }
+    }
+
+    &:active {
```

```suggestion
     &:active:not(:disabled):not([aria-disabled]) {
```
To be sure to avoid conflict with disabled state